community-membership.md

```diff
@@ -41,58 +41,24 @@ Members are expected to remain active contributors to the community.
     - Contributing to SIG, subproject, or community discussions (e.g. meetings, Slack, email discussion
       forums, Stack Overflow)
 - Subscribed to [kubernetes-dev@googlegroups.com]
+- Have read the [developer guide]
 - Actively contributing to 1 or more subprojects.
 - Sponsored by 2 reviewers. **Note the following requirements for sponsors**:
     - Sponsors must have close interactions with the prospective member - e.g. code/design/proposal review, coordinating
       on issues, etc.
     - Sponsors must be reviewers or approvers in at least 1 OWNERS file (in any repo in the Kubernetes GitHub
       organization)
     - Sponsors must be from multiple member companies to demonstrate integration across community.
-- Send an email to *kubernetes-membership@googlegroups.com* with:
-   - CC: your sponsors on the message
-   - Subject: `REQUEST: New membership for <your-GH-handle>`
-   - Body: Confirm that you have joined kubernetes-dev@googlegroups.com (e.g. `I have joined
-     kubernetes-dev@googlegroups.com`)
-   - Body: Your GitHub handle
-   - Body: GitHub handles of sponsors
-   - Body: List of contributions (PRs authored / reviewed, Issues responded to, etc)
+- **[Open an issue](https://github.com/kubernetes/org/issues/new?template=membership.md&title=REQUEST%3A%20New%20membership%20for%20%3Cyour-GH-handle%3E) against the kubernetes/org repo**
+   - Ensure your sponsors are @mentioned on the issue
+   - Complete every item on the checklist (you can preview the current version of the template [here](https://git.k8s.io/org/.github/ISSUE_TEMPLATE/membership.md))
```

Comment:
Style nit: here links.  Can we instead [preview the current version of the template](https://git.k8s.io/org/.github/ISSUE_TEMPLATE/membership.md)